2022 11 10 lps 164718 partial reindex categ 1672140761539

gustavolimav opened the pull request.

kristofloneck committed:
LPS-162735: Publish blog permission changes after auto-save

DiegoHu97 committed:
LPS-159432 Add new Date OOTB fragment

DiegoHu97 committed:
LPS-147897 Remove FF

JanisZhang committed:
LPS-165277 Remove DDMFormInstanceRecordWriterExtensions and use types instead.

JanisZhang committed:
LPS-165277 Apply usages and modify react component logic to accept a set, change the label to upper case to consist with the previous UI

JanisZhang committed:
LPS-165277 Convert method reference by using serviceTracker

JanisZhang committed:
LPS-165277 Remove DDMFormInstanceRecordWriterRegistryImplTest

kristofloneck committed:
LPS-168680:Making sure publish don't reset permission after initial entry creation

evibudai committed:
LPS-169318 The issue was caused by LPS-136334(61b5aee358cfc4748a50b0112b63a95562ded72c). Before LPS-136334, it used InlineSQLHelperUtil.replacePermissionCheck() to generate the sql(use ownerId). After LPS-136334, it used getPermissionWherePredicate() api, so ownerId was replaced by userId. This will cause performance issues.

kristofloneck committed:
LPS-168680: SF

AliciaGarciaGarcia committed:
LPS-170079 Expiring soon label

AliciaGarciaGarcia committed:
LPS-170079 buildLang

AliciaGarciaGarcia committed:
LPS-170079 extract logic to a display context

AliciaGarciaGarcia committed:
LPS-170079 add label warning if KB Article expires soon

AliciaGarciaGarcia committed:
LPS-170079 check the notification date weeks to show or not the new message

AliciaGarciaGarcia committed:
LPS-170079 only show new label if FF is active

oliv-yu committed:
LPS-172073 search-experiences-web: Remove unused CSS for blueprints search container

oliv-yu committed:
LPS-172073 search-experiences-web: Rename CSS variables based on Lexicon

oliv-yu committed:
LPS-172073 search-experiences-web: Remove need for a util CSS

Commit:
LPS-171507 Add virtual host and port parameters in object related endpoints

kristofloneck committed:
LPS-168680:Rename

oliv-yu committed:
LPS-172073 search-experiences-web: Categorize CSS files

oliv-yu committed:
LPS-172073 search-experiences-web: Rename 'builder' in CSS files

slnn committed:
LPS-171993 Remove the usage of Optional as return value in FileEntryAnalyticsReportsInfoItem._getUser(FileEntry fileEntry)

slnn committed:
LPS-171993 Remove the usage of Optional as return value in JournalArticleAnalyticsReportsInfoItem._getUser(JournalArticle journalArticle)

slnn committed:
LPS-171993 Remove the usage of Optional as return value in LayoutDisplayPageObjectProviderAnalyticsReportsInfoItemImpl._getThemeDisplayOptional() and rename to _getThemeDisplay()

slnn committed:
LPS-171993 Remove the usage of Optional as return value in LayoutAnalyticsReportsInfoItem._getThemeDisplayOptional() and rename to   _getThemeDisplay()

JhossephG committed:
LPS-168228 Add CanAddManyFieldsOnLayout Test

oliv-yu committed:
LPS-172073 search-experiences-web: Clarify classnames of modals on root level

oliv-yu committed:
LPS-172073 search-experiences-web: Separate modals into own component

oliv-yu committed:
LPS-172073 search-experiences-web: Clarify categorization and use underscore folder names

Commit:
LPS-171015 Add CompareUpgradedAndFreshDatabaseSchemas for all versions

Commit:
LPS-171507 Add tests to filter out object endpoints from other virtual instances

qz1010 committed:
LPS-171976 Add logic to check more case

qz1010 committed:
LPS-171976 Add testcase

qz1010 committed:
LPS-171976 Auto SF

evibudai committed:
LPS-169318 Fix failing test case

danvb112 committed:
LPS-171873 Remove unnecessary useEffects from ObjectFieldFormBase code

slnn committed:
LPS-171993 Remove the usage of Optional as return value in LayoutDisplayPageObjectProviderAnalyticsReportsInfoItemImpl. _getLayoutOptional(LayoutDisplayPageObjectProvider<?> layoutDisplayPageObjectProvider) and rename to  _getLayout(LayoutDisplayPageObjectProvider<?> layoutDisplayPageObjectProvider)

slnn committed:
LPS-171993 Apply

Commit:
LPS-166566 Add Interface to connect portal-search and search-experiences

Commit:
LPS-166566 Add code to redirect to textEmbeddingBackGroundTaskExecutor

Commit:
LPS-166566 Add dependence

Commit:
LPS-166566 Add code to reindex text Embedding

Commit:
LPS-166566 Implement execute method from backgroundtaskexecutor

qz1010 committed:
LPS-171976 Fix

qz1010 committed:
LPS-171976 Add testcase

qz1010 committed:
LPS-171976 Auto SF

liuxingye committed:
LPS-170542 Add object with validation file to dependencies

liuxingye committed:
LPS-170542 Add CanImportAndExportValidationStructure test

liuxingye committed:
LPS-160652 Add test for import object with action

Commit:
LPS-166566 Use each textEmbeddingDocumentContributors and fix services

pei-jung committed:
LPS-171650 (headless-admin-user-impl) - define schema for ticket

pei-jung committed:
LPS-171650 (headless-admin-user-impl) - add endpoints to retrieve a user's ticket

Commit:
LPS-166566 Refactor and Remove unecessary code

pei-jung committed:
LPS-171650 Build REST

slnn committed:
LPS-172518 Remove Optional usage in DefaultAssetDisplayPageFriendlyURLResolver's method body

slnn committed:
LPS-172518 SF, inline

slnn committed:
LPS-172518 Remove Optional usage in LinkedAssetEntryIdsUtil's method body

slnn committed:
LPS-172518 Remove Optional usage in LayoutPageTemplateStructureRelModelListener's method body

slnn committed:
LPS-172518 Remove Optional usage in FragmentEntryLinkModelListener's method body

slnn committed:
LPS-172518 Remove Optional usage in AssetDisplayLayoutUpgradeProcess's method body

slnn committed:
LPS-172518 Remove Optional usage in AssetDisplayPageEntryLocalServiceImpl's method body

slnn committed:
LPS-172518 SF, inline

Commit:
LPS-167249 SearchFacetRegistry: Convert method reference to ServiceTrackerList

Commit:
LPS-167249 Fix the test

slnn committed:
LPS-172518 Remove Optional usage in AssetTagStagedModelDataHandler's method body

victorg1991 committed:
LPS-172558 Allow indicating in the info item creator if the entity supports categorization and what scope uses

victorg1991 committed:
LPS-172558 Implement interface

victorg1991 committed:
LPS-172558 Show an alert if the form doesn't support categorization

victorg1991 committed:
LPS-172558 If the form has company scope make sure to only show vocabularies from global

victorg1991 committed:
LPS-172558 Language key

victorg1991 committed:
LPS-172558 buildLang

danvb112 committed:
LPS-172482 Add validation when there is no related entry

victorg1991 committed:
LPS-172586 Add type to the fragment renderer fragments

pei-jung committed:
LPS-171650 (headless-admin-user-impl) - implement new endpoints

pei-jung committed:
LPS-171650 (headless-admin-user-test) - enable test and implement test helpers

pei-jung committed:
LPS-171650 (headless-admin-user-impl) - apply permission check to the new endpoints

pei-jung committed:
LPS-171650 (headless-admin-user-test) - add tests for permission checks

victorg1991 committed:
LPS-172586 Add tags fragment renderer

victorg1991 committed:
LPS-172586 Use inert attribute instead of the css

DiegoHu97 committed:
LPS-159432 Add target blank to documentation link

kyle-miho committed:
LRQA-78905 Add automation for Liferay Home symbolic links

Commit:
LPS-171791 Add CanAccessDealFormViaBannerButton Test

ling-alan-huang committed:
LPS-171976 Rename

ling-alan-huang committed:
LPS-171976 SF

kristofloneck committed:
LPS-168680:Moving logics together.

ling-alan-huang committed:
LPS-171976 Remvoe unneeded variable

ling-alan-huang committed:
LPS-171976 SF, buy space

ling-alan-huang committed:
LPS-171976 More accurate

ling-alan-huang committed:
LPS-171976 SF, inline

Commit:
LPS-162335 Fix method _addRole

ling-alan-huang committed:
LPS-172446 Auto SF

ling-alan-huang committed:
LPS-172446 SF

victorg1991 committed:
LPS-172586 Update language key

victorg1991 committed:
LPS-172586 buildLang

jkappler committed:
LPS-172865 Adds new resource action for assigning/un-assigning utility pages

jkappler committed:
LPS-172865 Adds language key

jkappler committed:
LPS-172865 Build lang

jkappler committed:
LPS-172865 Adds action key

jkappler committed:
LPS-172865 Baseline

jkappler committed:
LPS-172865 Only let the user interact with the action if they have permissions to do so

p2kmgcl committed:
LPS-172874 Move app-hooks to app/hooks/app_hooks

p2kmgcl committed:
LPS-172875 Merge core and common directories

jkappler committed:
LPS-172865 Check for the permissions when setting default utility pages

jkappler committed:
LPS-172865 Adds mew method to unset a default utility page

jkappler committed:
LPS-172865 Build service

jkappler committed:
LPS-172865 Use new methods

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

veroglez committed:
LPS-168779 Use liferay-ui:tabs in info panel

sergiojimcos committed:
LPS-172697 Support 1toMObjectRelationshipERC in nested relationship node

slnn committed:
LPS-171993 Apply

veroglez committed:
LPS-168779 Remove unnecessary navigationItems

veroglez committed:
LPS-168779 Replace h4 header with h2

slnn committed:
LPS-171993 Apply

slnn committed:
LPS-171993 Apply

jkappler committed:
LPS-172865 Integration test for setting a default utility page with permissions

jkappler committed:
LPS-172865 Integration test for unsetting a default utility page with permissions

slnn committed:
LPS-171993 Apply

jkappler committed:
LPS-172865 Integration test for setting a default utility page without permissions

jkappler committed:
LPS-172865 Integration test for unsetting a default utility page without permissions

slnn committed:
LPS-172896 Remove usages of Stream/Optional in Method body for dxp module: document-library-opener

jkappler committed:
LPS-172865 Only let the user interact with the action if they have permissions to do so

jkappler committed:
LPS-172865 We really need to check if we have the permissions to delete an entry. Also, if it's marked as default, don't let it be deleted unless we have the specific permission to do so

jkappler committed:
LPS-172865 Integration test for deleting a default utility page with permissions

jkappler committed:
LPS-172865 Integration test for deleting a default utility page with no permissions

jkappler committed:
LPS-172865 Integration test for deleting a utility page with no assign permissions

Commit:
LPS-162335 - Fix text _assertRoles

Commit:
LPS-162335 SF

Commit:
LPS-162335 Fix the if logic

Commit:
LPS-172637 Expose the issue.

ces-totrinh committed:
LPS-172966 Avoid processing images when fileEntry is not an image

JanisZhang committed:
LPS-165277 Sem Ver

jkappler committed:
LPS-172869 Just like we do with fragments (entries and collections) and with layout page temples (entries and collections), calculate by hand. We remove spaces and change them for dashes

jkappler committed:
LPS-172869 Add integration tests

ling-alan-huang committed:
LPS-171976 SF, get* method always returns something

p2kmgcl committed:
LPS-172876 Use underscore for directory names

evibudai committed:
LPS-170303 Using 1.2.2.v20211118-0920-LIFERAY-CACHE.jar that contains LPS-108112, LPS-129450 and LPS-140601, instead of 1.2.2.v20211119-2358-LIFERAY-CACHED.jar that only contains LPS-140601

evibudai committed:
LPS-170303 Apply usage

sergiojimcos committed:
LPS-172697 Add test case

ling-alan-huang committed:
LPS-171976 SF

lfbesada committed:
LPS-173001 Allow editing the friendlyURL

joseabelenda committed:
LPS-162335 SF

carolmariaabb committed:
LPS-165277 The fileExtensions parameter is now an array of strings

carolmariaabb committed:
LPS-165277 Fix unit test

victorg1991 committed:
LPS-172103 Send the css, html and js as files

victorg1991 committed:
LPS-172103 Convert Render command to resource command to be able to send files

victorg1991 committed:
LPS-172103 Adapt classes to handle files

mbowerman committed:
LPS-173061 Wrap both sides of the expression in parentheses to ensure
that the predicate is parsed the same way as the OData Filter was

mbowerman committed:
LPS-173061 Add integration test

victorg1991 committed:
LPS-172586 baseline

oliv-yu committed:
LPS-172073 search-experiences-web: Organize shared components first in main scss file

Commit:
LPS-172637 Revert "LPS-172637 check companyAdmin by companyId"

This reverts commit 0a0436fc6471d5cf1f35bdd0a4afffd355fe22fc.

Hanlf committed:
LPS-168391 Update article to avoid translation flaky issue

Hanlf committed:
LPS-173117 Ignore warn until bug is fixed

jkappler committed:
LPS-172865 Use get directly here, since the portlet already handles this kind of exception

ces-totrinh committed:
LPS-172966 Update test

izaera committed:
LPS-172978 Revert previous rename

The rationale of this commit is that if we leave this key renamed, we are:

1. Breaking our convention of key->value naming
2. Making it impossible to define a category.remote-apps in the future because
   it would clash with Client Extensions
3. And, the most important: we are moving user's portlets from one category to
   another without telling them, which can create quite a lot of trouble and
   damage to our credibility.

So we revert it back to what it was, and make changes in code (see following
commits) to make new Client Extension portlets appear under the Client Extensions
category by default, then leave the older portlets as they were (untouched).

izaera committed:
LPS-172978 gradle buildLang

izaera committed:
LPS-172978 Remove unused key

izaera committed:
LPS-172978 gradle buildLang

izaera committed:
LPS-172978 Remove obsolete translations of ClientExtensionAdminPortlet

izaera committed:
LPS-172978 Add new category.client-extensions language key

izaera committed:
LPS-172978 gradle buildLang

izaera committed:
LPS-172978 Point Workspace project templates to new category

izaera committed:
LPS-172978 Use new category as default for Client Extensions portal app

izaera committed:
LPS-172978 Use new category in site initializers

izaera committed:
LPS-172978 Update Workspace tests

izaera committed:
LPS-172978 Update client extensions portal app tests

izaera committed:
LPS-172978 Update site initializer tests

izaera committed:
LPS-172978 Use new category in create_remote_app.sh

victorg1991 committed:
LPS-159432 Not needed

victorg1991 committed:
LPS-159432 Rename

bryceosterhaus committed:
LPS-168443 - Update npm-scripts to 47.12.0

bryceosterhaus committed:
LPS-168443 - Add allowed-non-global-dependencies to npmscripts.config

carolmariaabb committed:
LPS-171763 Revert "LPS-171354 validation for fields of type relationship to send empty value"

This reverts commit f7d0be51f4bf94f86b2d26116252233906761bdd.

carolmariaabb committed:
LPS-171763 Revert "LPS-171354 validate to empty fields not display an error"

This reverts commit 9251dd42207fa69772bfe23353a59ff6f4194f76.

victorg1991 committed:
LPS-172873 Take into account selPpi parameter, when reloading the product menu the ppid correspond to the ProductMenu Portlet instead of the current selected portlet

carolmariaabb committed:
LPS-171763 Send empty value instead of null when removing the relationship value

luisacmn committed:
LPS-172809 Add pageSize and modified type status

luisacmn committed:
LPS-172809 SF

Commit:
LPS-166566 Use companyLocalThread.forEachCompanyId

4lejandrito committed:
LPS-173173 Ommiting schema name when forcePredictableOperationId = true

ling-alan-huang committed:
LPS-154677 SF, remove nonexistent package

luisacmn committed:
LPS-172809 Modified condition for Approved deals

Commit:
LPS-166566 Change document builder import

ling-alan-huang committed:
LPS-171976 Only enable for master, Alan will revert when all violations are fixed on other branches

kiwm committed:
LPS-172789 [PRM] Change the test from end-date-six-month on yup.ts page

veroglez committed:
LPS-168761 Replace title with data-title to avoid reading the html with the screen reader

veroglez committed:
LPS-168761 Change the shortcut

veroglez committed:
LPS-168761 Replace aria-label with aria-labelledby and delete keys that will no longer be used

veroglez committed:
LPS-168761 Add aria-label in close button

Commit:
LPS-173201 The service context should have the update command

Commit:
LPS-171791 Add clickBannerLink Macro

Commit:
LPS-171791 Add BANNER_LINK Path

Commit:
LPS-171791 SF

veroglez committed:
LPS-168761 BuildLang

riccardo-ferrari committed:
LRAC-12756 Fix case-sensitive sort

xbrianlee committed:
LPS-168594 Use blacklist on upstream tests, but use enterprise.product.enterprise.search.enabled on release tests

joshchong committed:
LPS-173215 Update function to account for move to Chrome100

joshchong committed:
LPS-173217 Update path due to changes to markup

brianchandotcom committed:
LPS-172869 Match generated SQL

brianchandotcom committed:
LPS-172869 SF

brianchandotcom committed:
LPS-172869 SF

Commit:
LPS-172282 Remove redundant parameters

shuyangzhou committed:
LPS-173309 Avoid duplicated company index initialization.

Commit:
LPS-171015 Change build-test.xml to delete _x_companyId references when comparing DBs from diff versions

Commit:
LPS-169969 Delete master Data folder before rebuilding legacy DB to correctly run upgrade tests

ling-alan-huang committed:
LPS-171056 Auto SF

JoyceWang08211 committed:
LRQA-79120 Update case

JoyceWang08211 committed:
LRQA-79117 Update macro

peerkar committed:
LPS-172881 search-experiences-service: Fix wrong replacement sequence. Add dollar sign to avoid LPS-173239

liferay-translation committed:
LPS-77699 Update Translations

JessieZhao0714 committed:
LRQA-79172 Change the method of modifying custom field

victorg1991 committed:
LPS-172103 Adapt test

Commit:
Revert "LPS-172681 Restrict acces by URL and display error"

This reverts commit 4f161e430f0665605afd1512c383f4df6d574375.

ericthmoritsuka committed:
LPS-172704 adjusting badly formatted CSS

tinatian committed:
LPS-172637 Do not use system company Id (0) as default, we should use the companyId retrieved from current permission checker as default

brianchandotcom committed:
LPS-172697 Rename

brianchandotcom committed:
LPS-172103 SF

brianchandotcom committed:
LPS-173061 Sort

brianchandotcom committed:
LPS-151626 Rename refresh_sample.sh

itstali committed:
LPS-173196 Skip D2C Tests

brianchandotcom committed:
LPS-151626 add refresh_liferay.sh

brianchandotcom committed:
LRCI-3345

itstali committed:
LPS-173196 Skip D2C Tests SF

brianchandotcom committed:
LPS-170079 Wordsmith

brianchandotcom committed:
LPS-170079 Regen

brianchandotcom committed:
LPS-170079 as used

brianchandotcom committed:
LPS-173173 prep next

brianchandotcom committed:
LPS-173173 prep next

brianchandotcom committed:
LPS-172966 Sort

joseabelenda committed:
LPS-173268 Fix criteria to render file selector

brianchandotcom committed:
LPS-172518 Buy space

brookedalton committed:
Revert "LPS-167763 Avoid stream"

This reverts commit 5f4f9911d9decf5b52317e060adcdd95ecfd0d0b.

brookedalton committed:
Revert "LPS-167763 Display correct results when permission scope is changed in a role"

This reverts commit ebdfb993d7d89671fb70022660b89ef510ba944d.

Commit:
LPS-171015 Adding ci.retries.disabled property to run just once at CI

nkkwon committed:
LRQA-79198 Ignore failing a11ytool tests

gamerson committed:
LRQA-79070 simplify jsont files

gamerson committed:
LRQA-79070 regen

vicnate5 committed:
LPS-171015 Use latest 6.2 version

brianchandotcom committed:
LPS-172637 Rename

brianchandotcom committed:
artifact:ignore digital-signature-web 1.0.44 prep next

brianchandotcom committed:
artifact:ignore portal-search-rest-impl 1.0.19 prep next

brianchandotcom committed:
LPS-170303 prep next

brianchandotcom committed:
artifact:ignore portal-search-web 6.0.85 prep next

brianchandotcom committed:
artifact:ignore portal-security-permission-impl 4.0.19 prep next

brianchandotcom committed:
LRQA-79070 Regen

liferay-translation committed:
LPS-77699 Update Translations

Commit:
LPS-166566 Add Validation to textEmbeddingEnabled() and add companyId to query

Commit:
Playing with searchAfter
